### ISSUE TITLE
"Booting 'local image A'" -- Dell Inspiron 11 3180

### ISSUE THREAD

**Issue by bmlzootown** [arnoldthebat/chromiumos#293]
Tried booting up via both image A and B, neither work. The furthest I get is "Booting 'local image A'". I can boot into FydeOS and CloudReady just fine, but the Linux (beta) wouldn't work in one, and my microSD reader wouldn't work in the other (although it showed up in lsusb, but was nowhere to be found in /dev).

My device is a Dell Inspiron 11 3180 with an AMD E2-9000e, AMD Radeon R2 integrated graphics, 4GB of RAM, 32GB eMMC, RTS5129 (Realtek) microSD reader, and an Atheros AR9565 wireless card.

Any suggestions/help would be greatly appreciated!

**Comment by neverfind** [comment]
Hey, I have the same issue! Except, I have an A9 Processor and R5 Graphics.

**Comment by xeu100** [comment]
Yeah, I've come to the conclusion that it's Radeon graphics compatibility.

**Comment by neverfind** [comment]
Do you mean it’s now Radeon compatible?

**Comment by xeu100** [comment]
No, I mean the issue is radeon graphics compatibility. I'm using Radeon Vega 8 graphics and it just won't work. Neverware's Cloudready has the drivers but it's closed source.

**Comment by aliberti98** [comment]
i7-3630qm (Intel HD4000 and Nvidia GT740M) here, same problem. Any suggestions?
